types/papaparse/index.d.ts

```diff
@@ -104,6 +105,7 @@ export interface ParseConfig {
     skipEmptyLines?: boolean | 'greedy'; // default: false
     fastMode?: boolean; // default: undefined
     withCredentials?: boolean; // default: undefined
+    delimitersToGuess?: string[]; // default: [',', '\t', '|', ';', Papa.RECORD_SEP, Papa.UNIT_SEP]
```

Comment:
Since this option has finite possible values, I would recommend making a separate type so that editors can provide better hints:
```typescript
...
type GuessableDelimiters = ','| '\t'| '|'| ';' | Papa.RECORD_SEP | Papa.UNIT_SEP
...
    delimitersToGuess?: GuessableDelimiters[]
...
```

Reply:
Thank you for your detailed feedback. It's my first ever contribution so I really appreciate. 

I will make the required changes and re-submit.